ui/src/app/store/project/applications/applications.action.ts

```diff
@@ -0,0 +1,116 @@
+
+import { Application } from 'app/model/application.model';
+import { ProjectIntegration } from 'app/model/integration.model';
+import { Key } from 'app/model/keys.model';
+import { Variable } from 'app/model/variable.model';
+
+// Use to load fetched application in our app
+export class LoadApplication {
```

Comment:
s/LoadApplication/LoadApplicationAction
LoadApplicationAction

Reply:
No, try to be concise as possible and follow guidelines https://ngxs.gitbook.io/ngxs/recipes/style-guide#action-suffixes

Reply:
So, why ApplicationsState ? 
this style-guide is weird :/